nova/nova/models.py

```diff
@@ -32,45 +31,63 @@ class User(AbstractUser):
     USERNAME_FIELD = 'email'
     REQUIRED_FIELDS = ['created_at', 'date_of_birth', 'groups', 'first_name', 'last_name']
 
-
 class Article(models.Model):
     created_at = models.DateTimeField(auto_now_add=True)
 
-    # foreign key -> many to one
-    author = models.ForeignKey(User, on_delete=models.CASCADE)
+    ## foreign key -> many to one
+    author = models.ForeignKey('User', on_delete=models.CASCADE, default=0)
 
     title = models.CharField(max_length=100)
 
     content = models.CharField(max_length=3000)
 
     rating = models.FloatField(blank=True, default=0)
 
-    REQUIRED_FIELDS = ['created_at', 'title', 'author', 'content']
+    REQUIRED_FIELDS = ['created_at', 'title',  'content']
+
 
 
 class TradingEquipment(models.Model):
     type = models.CharField(max_length=30)
 
     name = models.CharField(max_length=50)
 
-    daily_prices = ArrayField(ArrayField(models.FloatField()))
+    daily_prices =  ArrayField(models.FloatField(), default=list)
 
-    current_price = models.FloatField()
+    current_price = models.FloatField(default=0)
 
     REQUIRED_FIELDS = ['type', 'name']
 
 
 class Comment(models.Model):
     created_at = models.DateTimeField(auto_now_add=True)
 
-    # article comment, foreignkey -> many to one
-    article = models.ForeignKey(Article, on_delete=models.CASCADE)
+    ## article comment, foreignkey -> many to one
+    article = models.ForeignKey('Article' , on_delete=models.CASCADE, null = True)
+
+    ## trading equipment comment
+    trading_eq = models.ForeignKey(TradingEquipment, on_delete=models.CASCADE, null=True)
+
+    author = models.ForeignKey('User', on_delete=models.CASCADE)
+
+    content = models.CharField(max_length = 300)
+
+    REQUIRED_FIELDS = ['created_at', 'author', 'content']
+
+
+class Parity(models.Model):
+    observed_at = models.DateTimeField(blank=True)
+
+    from_eq = models.ForeignKey(TradingEquipment, on_delete=models.CASCADE, blank=False, related_name='parity_from')
```

Comment:
I think this way of modeling is not the best way. Foreign exchange can be considered as a trading equipment. Instead of a stock name like "MSTF", we'll have "EUR_TRY"

Reply:
So you suggest that there shouldn't be two equipments defining one parity and we should concatenate them to one string field?

Reply:
Aren't currencies by themselves, say EUR, a trading equipment too? When we implement trading equipment predictions, should users see specific data for a single currency or should they be evaluated as pairs?